2020-10-10 20:08:17,998 INFO: EmalTest startup [in /home/william/workspace/EmalTest/EmalTest/app/__init__.py:47]
2020-10-10 20:17:52,124 INFO: EmalTest startup [in /home/william/workspace/EmalTest/EmalTest/app/__init__.py:47]
2020-10-10 20:17:58,434 ERROR: Exception on /index [GET] [in /home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/william/workspace/EmalTest/EmalTest/app/routes.py", line 23, in index
    form = JoinTestForm()
NameError: name 'JoinTestForm' is not defined
2020-10-10 20:18:22,409 INFO: EmalTest startup [in /home/william/workspace/EmalTest/EmalTest/app/__init__.py:47]
2020-10-10 20:21:47,879 INFO: EmalTest startup [in /home/william/workspace/EmalTest/EmalTest/app/__init__.py:47]
2020-10-10 20:21:52,391 ERROR: Exception on /index [POST] [in /home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/william/workspace/EmalTest/env/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/william/workspace/EmalTest/EmalTest/app/routes.py", line 27, in index
    print(form.option.value)
AttributeError: 'BooleanField' object has no attribute 'value'
2020-10-10 20:22:44,329 INFO: EmalTest startup [in /home/william/workspace/EmalTest/EmalTest/app/__init__.py:47]
2020-10-10 20:24:22,377 INFO: EmalTest startup [in /home/william/workspace/EmalTest/EmalTest/app/__init__.py:47]
2020-10-10 20:24:46,394 INFO: EmalTest startup [in /home/william/workspace/EmalTest/EmalTest/app/__init__.py:47]
